BCP195bis/draft-ietf-uta-rfc7525bis.md

```diff
@@ -344,18 +346,32 @@ Implementers should note that compression at higher protocol levels can allow an
 ## TLS Session Resumption
 {: #rec-resume}
 
-If TLS session resumption is used in the context of TLS 1.2, care ought to be taken to do so safely. In particular, when using session tickets {{?RFC5077}}, the resumption information MUST be authenticated and encrypted to prevent modification or eavesdropping by an attacker. Further recommendations apply to session tickets:
+Session resumption drastically reduces the number of TLS handshakes and thus is an essential
+performance feature for most deployments.
+
+Stateless session resumption with session tickets is a popular strategy. It is specified in
+{{?RFC5077}} for TLS 1.2,
+and the equivalent PSK-based mechanism for TLS 1.3, Section 4.6.1 of {{RFC8446}}.
+When it is used, the resumption information MUST
+be authenticated and encrypted to prevent modification or eavesdropping by an attacker.
+Further recommendations apply to session tickets:
 
 * A strong cipher suite MUST be used when encrypting the ticket (as least as strong as the main TLS cipher suite).
 
 * Ticket keys MUST be changed regularly, e.g., once every week, so as not to negate the benefits of forward secrecy (see {{sec-pfs}} for details on forward secrecy).
 
 * For similar reasons, session ticket validity SHOULD be limited to a reasonable duration (e.g., half as long as ticket key validity).
 
-
-
 Rationale: session resumption is another kind of TLS handshake, and therefore must be as secure as the initial handshake. This document ({{detail}}) recommends the use of cipher suites that provide forward secrecy, i.e. that prevent an attacker who gains momentary access to the TLS endpoint (either client or server) and its secrets from reading either past or future communication. The tickets must be managed so as not to negate this security property.
 
+TLS 1.3 provides the powerful option of forward secrecy even within a long-lived connection
+that is periodically resumed. Section 2.2 of {{RFC8446}} recommends that clients SHOULD
+send an "key_share" when initiating session resumption.
+In order to gain forward secrecy, we recommend that server implementations SHOULD
```

Comment:
I would delete "we recommend"

Reply:
I used the term specifically here to clarify that this is a new recommendation, in contrast with the client-side recommendation which comes from RFC8446.

Reply:
Maybe: `[...], it is RECOMMENDED that server implementations respond with a "key_share"` ?

Reply:
This document RECOMMENDS that... ?